Source/WebCore/bindings/js/SerializedScriptValue.cpp

```diff
@@ -800,7 +806,9 @@ class CloneSerializer : CloneBase {
     static SerializationReturnCode serialize(JSGlobalObject* lexicalGlobalObject, JSValue value, Vector<RefPtr<MessagePort>>& messagePorts, Vector<RefPtr<JSC::ArrayBuffer>>& arrayBuffers, const Vector<RefPtr<ImageBitmap>>& imageBitmaps,
 #if ENABLE(OFFSCREEN_CANVAS_IN_WORKERS)
             const Vector<RefPtr<OffscreenCanvas>>& offscreenCanvases,
+            Vector<RefPtr<OffscreenCanvas>>& inMemoryOffscreenCanvases,
 #endif
+        Vector<RefPtr<MessagePort>>& inMemoryMessagePorts,
```

Comment:
Slightly inconsistent indenting here. When we end up with these large packages of parameters, I wonder if we might need some kind of structure instead of a parameter list, anyway. Should be fine for now, but `#if` and parameter lists don’t combined well.